client/state/data-layer/wpcom/jetpack-install/index.js

```diff
@@ -40,15 +35,26 @@ export const handleResponse = ( { url }, data ) => {
 	if ( data.status ) {
```

Comment:
On reflection, I don't think this check serves any purpose now. Any non-error return from the API should be treated as success.